plumbum/machines/session.py

```diff
@@ -72,7 +72,8 @@ class SessionPopen(PopenAddons):
     """A shell-session-based ``Popen``-like object (has the following attributes: ``stdin``,
     ``stdout``, ``stderr``, ``returncode``)"""
 
-    def __init__(self, proc, argv, isatty, stdin, stdout, stderr, encoding):
+    def __init__(self, host, proc, argv, isatty, stdin, stdout, stderr, encoding):
```

Comment:
Wouldn't it be nicer to add `host` to the end, rather than the beginning, so as to not shift all the arguments? I'd be fine to make it keyword only, too. ;)

```suggestion
    def __init__(self, proc, argv, isatty, stdin, stdout, stderr, encoding, *, host):
```